ds3-sdk-integration/src/test/java/com/spectralogic/ds3client/integration/BucketIntegration_Test.java

```diff
@@ -103,6 +104,40 @@ public void deleteBucket() throws IOException, SignatureException {
     }
 
     @Test
+    public void getObjects() throws IOException, SignatureException, URISyntaxException, XmlProcessingException {
+        final String bucketName = "test_get_objs";
+        try {
+            client.putBucket(new PutBucketRequest(bucketName));
+            Util.loadBookTestData(client, bucketName);
+
+            HeadObjectResponse headResponse = client.headObject(new HeadObjectRequest(
+                    bucketName, "beowulf.txt"));
+            assertThat(headResponse.getStatus(),
+                    is(HeadObjectResponse.Status.EXISTS));
+
+            final GetObjectsResponse response = client
+                    .getObjects(new GetObjectsRequest().withBucket("test_get_objs"));
+
+            assertFalse(response.getS3ObjectList().getObjects().isEmpty());
+            assertThat(response.getS3ObjectList().getObjects().size(), is(4));
+            assertTrue(s3ObjectExists(response.getS3ObjectList().getObjects(), "beowulf.txt"));
+
+        } finally {
+            Util.deleteAllContents(client,bucketName);
+        }
+
+    }
+
+    private boolean s3ObjectExists(List<S3Object> objects, final String fileName) {
+        for(S3Object o : objects) {
```

Comment:
`objects` is missing a `final`, as is `o` and I would change the name of `o` into something more descriptive like `obj` or something.